generators/server/templates/src/main/java/package/service/AuditEventService.java.ejs

```diff
@@ -38,16 +45,36 @@ import java.util.Optional;
 @Transactional<% } %>
 public class AuditEventService {
 
+    private final Logger log = LoggerFactory.getLogger(AuditEventService.class);
+
+    private final ApplicationProperties.AuditEvents auditEvents;
+
     private final PersistenceAuditEventRepository persistenceAuditEventRepository;
 
     private final AuditEventConverter auditEventConverter;
 
     public AuditEventService(
         PersistenceAuditEventRepository persistenceAuditEventRepository,
-        AuditEventConverter auditEventConverter) {
+        AuditEventConverter auditEventConverter, ApplicationProperties applicationProperties) {
```

Comment:
You need to substitute by JhipsterProperties

Reply:
Done.